modules/alb/main.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_lb" "this" {
+  name               = "${var.namespace}-alb"
+  subnets            = var.subnets_ids
+  load_balancer_type = "application"
+  security_groups    = var.security_group_ids
+
+  enable_deletion_protection = true
+  drop_invalid_header_fields = true
+
+
+  access_logs {
+    bucket  = "${var.namespace}-alb-logs"
+    enabled = true
+  }
+}
+
+resource "aws_lb_target_group" "this" {
+  name        = "${var.namespace}-alb-tg"
+  port        = var.app_port
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+  target_type = "ip"
+
+  health_check {
+    healthy_threshold   = 3
+    interval            = 5
+    protocol            = "HTTP"
+    matcher             = "200-299"
+    timeout             = 3
```

Comment:
are you going to run ruby app? if yes, these numbers can be low https://nimble-co.slack.com/archives/CDZAY1Z4L/p1686799115700509?thread_ts=1686795546.965199&cid=CDZAY1Z4L

Reply:
I'm going to use a NodeJS app, not Rails, so let's test it out first; if there are any issues, I will change it as per Xavier's suggestion.